app/scripts/modules/amazon/src/help/amazon.help.ts

```diff
@@ -114,6 +114,23 @@ const helpContents: { [key: string]: string } = {
       <p>Ensures that the capacity of this server group has not changed in the background (i.e. due to autoscaling activity).</p>
       <p>If the capacity has changed, this resize operation will be rejected.</p>`,
   'aws.tagImage.consideredStages': `Limit which previous stages will be considered when locating AMI's to tag. If left unchecked, AMI's generated by any upstream stage will be tagged.`,
+  'aws.loadBalancer.redirect.host': `The hostname. This component is not percent-encoded. The hostname can contain #{host}.`,
+  'aws.loadBalancer.redirect.path': `The absolute path, starting with the leading "/". This component is not percent-encoded. The path can contain #{host}, #{path}, and #{port}.`,
+  'aws.loadBalancer.redirect.port': `The port. You can specify a value from 1 to 65535 or #{port}.`,
+  'aws.loadBalancer.redirect.protocol': `The protocol. You can specify HTTP, HTTPS, or #{protocol}. You can redirect HTTP to HTTP, HTTP to HTTPS, and HTTPS to HTTPS. You cannot redirect HTTPS to HTTP.`,
+  'aws.loadBalancer.redirect.query': `The query parameters, URL-encoded when necessary, but not percent-encoded. Do not include the leading "?", as it is automatically added. You can specify any of the reserved keywords.`,
+  'aws.loadBalancer.redirect.statusCode': `The HTTP redirect code. The redirect is either permanent (HTTP 301) or temporary (HTTP 302).`,
+  'aws.loadBalancer.redirect': `<p>A URI consists of the following components: protocol://hostname:port/path?query. You must modify at least one of the following components to avoid a redirect loop: protocol, hostname, port, or path. Any components that you do not modify retain their original values.
```

Comment:
We should add markdown support to `HelpField.tsx` and start using markdown

Reply:
Man, you're volunteering for so much stuff, how nice of you...